invenio_rdm_records/config.py

```diff
@@ -559,6 +559,12 @@ def lock_edit_published_files(service, identity, record=None, draft=None):
 RDM_RECORDS_MAX_MEDIA_FILES_COUNT = 100
 """Max amount of media files allowed to upload in the deposit form."""
 
+RDM_MEDIA_FILES_DEFAULT_QUOTA_SIZE = 10**10
```

Comment:
Based on [`FILES_REST_DEFAULT_QUOTA_SIZE = 10**10`](https://github.com/inveniosoftware/invenio-app-rdm/blob/7d845a427797388e3d1dd3a0ea8820d93e478275/invenio_app_rdm/config.py#L314).
Should I add `RDM_FILES_DEFAULT_QUOTA` here as well? Right now it is being copied here: 
https://github.com/inveniosoftware/invenio-rdm-records/blob/2ed84dae6287befec91e1c0395fa0cfd8788426b/invenio_rdm_records/ext.py#L130

Reply:
Yeah, we should have both variations in the config here.